ganache-with-protocol/build-ganache.sh

```diff
@@ -0,0 +1,90 @@
+git clone -b streamflow https://github.com/livepeer/protocol.git
+cd protocol
+echo "Setting devenv specific protocol parameters"
+migrations="/protocol/migrations/migrations.config.js"
+sed -i 's/roundLength:.*$/roundLength: 50,/' $migrations
+sed -i 's/unlockPeriod:.*$/unlockPeriod: 50,/' $migrations
+sed -i 's/numActiveTranscoders:.*$/numActiveTranscoders: 50,/' $migrations
+sed -i 's/numTranscoders:.*$/numTranscoders: 100,/' $migrations
+
+cat << EOF > /protocol/truffle.js
+require("babel-register")
+require("babel-polyfill")
+
+module.exports = {
+    networks: {
+        development: {
+            host: "localhost",
+            port: 8545,
+            network_id: "54321", // Match any network id
+            gas: 8000000
+        }
+    },
+    compilers: {
+        solc: {
+            version: "0.5.11",
+            parser: "solcjs",
+            settings: {
+                optimizer: {
+                    enabled: true,
+                    runs: 200
+                }
+            }
+        }
+    }
+};
+EOF
+
+echo "Installing local dev version of /protocol/scripts/unpauseController.js"
+
+cat <<EOF > /protocol/scripts/unpauseController.js
+const Controller = artifacts.require("Controller")
+
+module.exports = async cb => {
+    const controller = await Controller.deployed()
+    await controller.unpause()
+    cb()
+}
+EOF
+
+echo "npm install"
+npm install --unsafe-perm
```

Comment:
Does this not work without `--unsafe-perm`?

Reply:
It didn't initially, not sure why but it does now.

Reply:
fixed in 25e0ca1